src/styles/_fonts.scss

```diff
@@ -0,0 +1,9 @@
+// PF4 needed config for fonts
+$pf-global--disable-fontawesome: true;
+$pf-global--font-path: '@patternfly/patternfly/assets/fonts/';
+
+// This is necessary to have icons into Graph edge
+@font-face {
+  font-family: 'pficon';
+  src: url(../../node_modules/@patternfly/patternfly/assets/pficon/pficon.woff2);
```

Comment:
Do we want to reference this from under node_modules or should we copy this under src in package.json and use a local  path?

Reply:
Personally, I think this is fine the way it is.

Reply:
I am being consistent with other places where we load PF assets in the same way.
https://github.com/kiali/kiali-ui/blob/82f39359463a45f1a42b34a80e02be6d701ee22a/src/styles/index.scss#L10

I am not very familiar with the way you are suggesting. And copying assets from dependency to our source makes me think that we will be more error-prone in next pf upgrades.

Reply:
This way is fine!